adversarial_apps/README.md

```diff
@@ -34,3 +34,11 @@ You can check out [the Next.js GitHub repository](https://github.com/vercel/next
 The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
 
 Check out our [Next.js deployment documentation](https://nextjs.org/docs/app/building-your-application/deploying) for more details.
+
+## Linting
+
+It's recommended to install an ESLint extension for these to be handled during development.
```

Comment:
Need to change this to say and be more accurate: ESLint comes provided as part of the required packages for application. You must use that version of ESLint. When you install the packages as seen above in README you will have ESLint and can run the following command:

Reply:
Double check the lint to have it pass all linting checks with recent update to action

Reply:
They Validate now. I excluded `node_module`s from the black check since we are `.gitignoring` those files anyway.